websites/docs-smoke-test/src/content/components/cards.mdx

```diff
@@ -0,0 +1,203 @@
+---
+title: Cards
+---
+
+ * The actual card content is markdown
+ * Title have no linkable / hover anchor like section headings
+ * Titles never become entries in the page index nav like section headings
+ * Either an icon or an image path can be set, but not both.
+   * To start with, Images are _not_ supported for free-form in-content cards, only in places that are fully designed and implemented by developers. An Image is sized into the width of the card.
+   * Icons are supported anywhere but only those that are officially provided by the kit. An icon is sized into a fixed 48x48px square.
+ * Cards can occur in two interaction variants, "active" (visually elevated) and "flat" (visually an outline only).
+ * Cards must be embedded into a wrapper component that defines the responsive sizes (two aside / three aside) and parameters that must be identical in a give row or grid (active card area, the heading size)
+
+# "Cards" Wrapper
+
+ Properties (for authors):
+ * `active` (boolean): if set, the whole area of all child cards is a link, the card is elevated and has a hover effect if a link url is set.
```

Comment:
I personally prefer to have the functional goal ("make the whole card clickable") in all the content files and not the current visual style so I invented "active" instead of "raised". 

The reason for that mode is not the look (we would have preferred a flat look but it's not clear enough on mobile then that some cards are an active area and others aren't) but to have a large tappable area. 

"clickable" ?  "interactive" ? "linkCard"?     I could live with all these as long as it's not the visual style but the behavior that's indicated. 

for example Semantic UI just calls it a "link card" and you trigger it by putting a "link" class.